### DISCUSSION TITLE
Copy historical data to new install

### DISCUSSION THREAD

**Discussion by laferrierem** [jprochazka/adsb-receiver#606] [Q&A]
Hello Everyone.

     Is it possible to copy over the data from a current install to a new one? I have a few receivers I have been running since this project started and they are in need of updating. However, I would like to keep the data and graphs, maps etc. from the existing setups.

      Thank you.
      
      
![image](https://github.com/user-attachments/assets/819ae386-9878-402a-9d41-334a4c7a91d9)

**Comment by jprochazka** [comment]
It should be as simple as copying the RRD files over to the new installation(s).
The files reside in the directory /var/lib/collectd/rrd.

If you already did this and are having issues, can you do me a favor and screen shot the directory structure of your old rrd files and run one of the graph generation jobs manually and paste the output here?

`bash ~/adab-receiver/build/graphs/make-collectd-graphs.sh 1h`

**Reply by laferrierem** [reply]
pi@PI:~/adsb-receiver/build/portal/graphs $ dir
dump1090.db  dump1090.py  dump1090.pyc	DUMP1090-TOOLS_LICENSE.txt  make-collectd-graphs.sh  make-collectd-graphs.sh.save
pi@PI:~/adsb-receiver/build/portal/graphs $ sudo bash make-collectd-graphs.sh 1h
10936 (process ID) old priority 0, new priority 5
577x298
577x298
577x298
577x298
1107x284
577x298
577x298
577x298
577x298
577x298
577x298
577x298
577x298
577x298
1107x284
577x298
577x298
577x298
577x298
pi@PI:~/adsb-receiver/build/portal/graphs $

**Reply by jprochazka** [reply]
Looks good. Do not see any errors. Are the graphs showing?

**Comment by laferrierem** [comment]
Hello Joseph.

     Once again, thank you for taking the ADSB receiver project back up and
updating things. This is a cool project that I have been sharing with in
the amateur radio community. Its easy to grab a Pi, a USB receiver and we
can cut our antennas or buy some nice off the shelf ones to get going.

      The data is from the old version of your receiver. I used to have a
link with instructions on how to copy it.

      Thank you again.

**Reply by jprochazka** [reply]
This might be the link to the documentation you mentioned.
https://github.com/jprochazka/adsb-receiver/wiki/Data-Backup-and-Restoration-Procedures

**Reply by jprochazka** [reply]
I have another person having rrd data issues but not sure if they are related being his data is from a Graphs1090 installation. My problem is the receiver I had running back before the downtime is no longer at my disposal. Although the code has not really changed, and my current receiver is running fine I cannot discount the fact there might be an issue which needs to be addressed.

I am going to suggest as I did in the discussion linked below. If you are still having issues and do not mind sharing the data contained within the rrd databases feel free to hit me up. I will set up a way for you to share these files if you are not comfortable sharing them here.

The possible related discussion can be found here: https://github.com/jprochazka/adsb-receiver/discussions/604

**Reply by laferrierem** [reply]
That would be great! I was just on here and was going to try copying thing over again this weekend. I can ZIP up an image or two if needed. What would be the best way to get these to you? FTP? RAR & email etc.?

Its great to see the project moving forward again!
